### PR TITLE
style: Final One Click Clusters Updates

### PR DESCRIPTION
**Pull request by hmorris3293**
## Description 📝
- Added icon for Galera
- Cluster apps have been added to the New Apps section
- Cluster app detail drawer displays Cluster in the title and links to a Cluster doc under the Guides section
    - The guide link will 404 bc it's not published yet
- The Setup section heading title now displays Cluster as well
- Hourly cluster price is rounded to 3 decimal places

## Preview 📷
image
image
image

## How to test 🧪
Go to Marketplace and Click through the Cluster and non-cluster apps
```
yarn test AppDetailDrawer
```
```
yarn test packages/manager/src/features/linodes/LinodesCreate/utilities.test.ts
```

### CODE REVIEW

**Review by dwiley-akamai — COMMENTED** [review]
Seeing two failures in `utilities.test.ts`:

Screenshot 2023-03-15 at 4 32 43 PM

File: packages/manager/src/features/linodes/LinodesCreate/SelectionCardWrapper.tsx [hunk 1]
```diff
@@ -45,6 +45,7 @@ interface Props {
   iconUrl: string;
   id: number;
   label: string;
+  clusterLabel: string | null;
```
Comment (dwiley-akamai) [inline packages/manager/src/features/linodes/LinodesCreate/SelectionCardWrapper.tsx:48]: When would this be `null`? Looks like `decodedLabel` gets passed as this from `AppPanelSection.tsx`, and that is a `string`.

I only glanced at the [docs](https://github.com/mathiasbynens/he#hedecodehtml-options) re: `decode` but didn't see anything about it returning `null`.

**Review by mjac0bs — COMMENTED** [review]
Same feedback as Dajahi with the failed test. But otherwise the cluster apps look good with the relevant titles, added icon, and spot in the new apps list, all as described.

**Review by dwiley-akamai — APPROVED** [review]
Functionality LGTM, left one question/comment re: a type

**Review by mjac0bs — APPROVED** [review]

File: packages/manager/src/features/linodes/LinodesCreate/SelectionCardWrapper.tsx (hunk 1 above)
Comment (hana-linode) [reply packages/manager/src/features/linodes/LinodesCreate/SelectionCardWrapper.tsx:48]: Good catch, I forgot to clean this up 🔍

**Pull request merged**